Description: lp#1085169 -- hotfix to get menus to work e.g. in print preview
Author: Bjoern Michaelsen <bjoern.michaelsen@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1085169
Forwarded: no
Last-Update: 2013-04-16

---
--- libreoffice-4.0.2.orig/vcl/source/window/menu.cxx
+++ libreoffice-4.0.2/vcl/source/window/menu.cxx
@@ -1120,6 +1120,7 @@ void Menu::ImplSelect()
 
 void Menu::Select()
 {
+    aActivateHdl.Call( this );
     ImplMenuDelData aDelData( this );
 
     ImplCallEventListeners( VCLEVENT_MENU_SELECT, GetItemPos( GetCurItemId() ) );
